credentials/version.go

```diff
@@ -1,4 +1,13 @@
 package credentials
 
-// Version holds a string describing the current version
-const Version = "0.6.4"
+var (
+	// Package is filled at linking time
+	Package = "github.com/docker/docker-credential-helpers"
+
+	// Version holds the complete version number. Filled in at linking time.
+	Version = "v0.0.0+unknown"
+
+	// Revision is filled with the VCS (e.g. git) revision being used to build
+	// the program at linking time.
+	Revision = ""
```

Comment:
Are we already at a point where we could use Go's built-in features for this?

If so; can they still be overridden?

Reply:
Hum good idea we should look at this in a follow-up